app/models/comment.rb

```diff
@@ -0,0 +1,14 @@
+class Comment < ApplicationRecord
+  self.table_name = "submissions"
+
+  belongs_to :submitter, class_name: :User, foreign_key: :user_id
+  belongs_to :post
+
+  # scope :comments, -> { where(post_id: !nil) }
```

Comment:
remove this?

Reply:
At first I wrote it in, just in anticipation that I'd probably use it, but when I was getting the PR ready, I couldn't remember if I'd tested out how it was working, so left it as a comment.